tests/notificationprofile/test_views.py

```diff
@@ -74,29 +92,213 @@ def test_notification_profile_can_update_timeslot_without_changing_pk(self):
                 "active": self.notification_profile1.active,
             },
         )
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.status_code, status.HTTP_200_OK, response.data)
         self.assertEqual(response.data["pk"], profile1_pk)
         self.assertEqual(NotificationProfile.objects.get(pk=profile1_pk).timeslot.pk, self.timeslot2.pk)
 
+
+@tag("API", "integration")
+class NotificationIncidentViewTests(APITestCase):
+    def setUp(self):
+        disconnect_signals()
+        user1 = PersonUserFactory()
+
+        self.user1_rest_client = APIClient()
+        self.user1_rest_client.force_authenticate(user=user1)
+
+        source_type = SourceSystemTypeFactory(name="nav")
+        source1_user = SourceUserFactory(username="nav1")
+        self.source1 = SourceSystemFactory(name="NAV 1", type=source_type, user=source1_user)
+
+        source_type2 = SourceSystemTypeFactory(name="type2")
+        source2_user = SourceUserFactory(username="system_2")
+        self.source2 = SourceSystemFactory(name="System 2", type=source_type2, user=source2_user)
+
+        self.incident1 = StatelessIncidentFactory(source=self.source1)
+        incident2 = StatelessIncidentFactory(source=self.source2)
+
+        timeslot1 = TimeslotFactory(user=user1, name="Never")
+        filter1 = FilterFactory(
+            user=user1,
+            name="Critical incidents",
+            filter_string=f'{{"sourceSystemIds": [{self.source1.pk}]}}',
+        )
+        self.notification_profile1 = NotificationProfileFactory(user=user1, timeslot=timeslot1)
+        self.notification_profile1.filters.add(filter1)
+
+    def teardown(self):
+        connect_signals()
+
+    def test_can_get_all_incidents_of_notification_profile(self):
+        response = self.user1_rest_client.get(
+            path=f"/api/v2/notificationprofiles/{self.notification_profile1.pk}/incidents/"
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK, response.data)
+        self.assertEqual(len(response.data), 1)  # 1, not 2
+        self.assertEqual(response.data[0]["pk"], self.incident1.pk)
+
+
+@tag("API", "integration")
+class MediumViewTests(APITestCase):
+    def setUp(self):
+        disconnect_signals()
+        user1 = PersonUserFactory()
+
+        self.user1_rest_client = APIClient()
+        self.user1_rest_client.force_authenticate(user=user1)
+
+    def teardown(self):
+        connect_signals()
+
+    def can_get_json_schema_for_email(self):
```

Comment:
Is this a test? Then I assume it's not being run, due to the missing `test_` prefix:
```suggestion
    def test_can_get_json_schema_for_email(self):
```